course-backend/package.json

```diff
@@ -1 +1,14 @@
-{}
+{
+  "dependencies": {
+    "@nestjs/jwt": "^10.0.3",
```

Comment:
**issue:** something seems wrong here, you have very small number of depedencies here, please let's make a call to sync on this